.gitignore

```
@@ -18,3 +19,9 @@ testfm.egg-info/
 #Packaging
 build/
 dist/
+
+# Dynaconf local files
+/.secrets.yaml
+/.secrets_*.yaml
+/conf/*
+!/conf/testfm.sample.yaml
```

```suggestion
/conf/testfm.local.yaml
```
Should be enough! You will have max one YAML if I am not wrong.

---

local yaml is being ignored by git already under `/conf/*`,  and changes to sample yaml must be excluded by git here.

---

Got that, I mean to say to say that line you dont need two lines, it can fit into one as well :)

Not a blocking comment though.